README.md

```diff
@@ -22,12 +22,21 @@ from [kcdc.info](http://www.kcdc.info/), and returns results about these talks.
 
 #### Running
 
+**Prerequisites:**
+* [Maven 3.x](https://maven.apache.org/index.html)
+* [Java 8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+
 Assemble the service:
 
 ```bash
 mvn clean package
 ```
 
+Optionally, you can add `-DskipTests` if you notice test failures when running `mvn clean package`:
```

Comment:
I would prefer to not advertise anyone to skip tests when building locally (as this is hinting other problems will probably still occur on their setup). Shall we just include a link here for downloading the pre-built JAR for running the class (if they don't have the pre-reqs)?